frontend/public/components/utils/use-url-hook.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import { useCallback, useState } from 'react';
+import { usePoll, useSafeFetch } from '.';
```

Comment:
plaese dont import from index file, it causes circular deps

Reply:
fixed